plugins/shared/catalog/testing.go

```diff
@@ -1,3 +1,5 @@
+// +build !release
```

Comment:
I don't think this is necessary. It shouldn't be imported by anything in the build, only tests. None of our other testing files do this